build.gradle

```diff
@@ -189,8 +181,21 @@ static def addPomDetails(MavenPom pom, String name, String description) {
     }
 }
 
+static def getVersionType(String version) {
```

Comment:
I don't think this can be static, since it relies on `version`.

Reply:
what do you mean, sorry. This is dependent on `version` taken as parameter, not as global variable

Reply:
I think it is clearer.
I personally do not like it also in the case where `version` might not be set before before (because of developer mistake) and this would trigger a RELEASE artifact.